src/libponyrt/actor/actor.h

```diff
@@ -61,6 +61,7 @@ typedef struct pony_actor_t
   size_t muted; // 4/8 bytes
   // internal flags are only ever accessed from a single scheduler thread
   uint8_t internal_flags; // 4/8 bytes (after alignment)
+  uint32_t live_events;
```

Comment:
minor thing.. might be worth renaming this to be more explicit as `live_asio_events` instead..

Reply:
excellent point

Reply:
updated.